resources/monitoring/templates/prometheus/kyma-rules/kyma-rules.yaml

```diff
@@ -75,3 +75,13 @@ spec:
         severity: warning
       annotations:
         message: Pod {{`{{`}} $labels.namespace {{`}}`}}/{{`{{`}} $labels.pod {{`}}`}} ({{`{{`}} $labels.container {{`}}`}}) is OOMKilled for 5 minutes.
+  - name: prometheus-rules
+    rules:
+    - alert: ScrapeLimitForTargetExceeded
+      annotations:
+        description: One or more targets in the job monitoring-prometheus-istio-server have scrape limit exceeded
+        summary: Scrape limit for one or more targets is exceeded.
+      expr: increase(prometheus_target_scrapes_exceeded_sample_limit_total{job="monitoring-prometheus-istio-server"}[1m]) > 1
```

Comment:
Why the value should be greater than 1? It should fire if it is greater than 0.

Reply:
fixed.